### ISSUE TITLE
Profile folder copying issue on windows 10: problem still present

### ISSUE THREAD

**Issue by erotavlasme** [brave/brave-browser#8069]
Hi,

I already opened an [issue](https://github.com/brave/brave-browser/issues/7528) about copying profile folder on windows 10 1909 64 bit. The problem is still present.
If I manually copy %USERPROFILE%\AppData\Local\BraveSoftware\Brave-Browser\User Data\Default folder or the entire %USERPROFILE%\AppData\Local\BraveSoftware\Brave-Browser\ folder some preference are stored (theme, some settings) while other not (extensions, default search engine, enabled spelling languages and home button customized options). When I enter into settings, brave says that it found wrong configuration and ask me if I want to restore the default settings.
If I repeat the same procedures under GNU/linux, it works well and all setting and extension are present.

1. Copy the profile folder Brave-Browser\User Data\Default or the entire folder Brave-Browser.
2. On a different machine or virtual machine, replace the folder with Brave-Browser\User Data\Default or the entire folder Brave-Browser.
3. Start brave.

My extensions are: privacy possum, privacy badger, decentraleyes, cookies autodelete, terms of service, and country flag and IP.

Note: if I create a new user on the same windows installation instance it works perfectly and even the extensions are maintained. Whereas if I change the windows installation instance, it behaves as I described above.

The issue is always reproducible. Brave Version 1.2.43 Chromium: 79.0.3945.130 (Official Build) (64-bit).

**Comment by jDally987** [comment]
Yeah I’m getting this too. Did a clean install of Windows and copied over my chrome/brave UserData folders totally forgetting that Brave’s is broken for some reason.

I’ll probably just dig in the sqlite db’s and manually copy over the search engines & everything myself, but what the heck is the main bug here? There must be an encrypted token in one or some of the files specific to the original Windows installation.... but idk. Really wish there was better documentation on this, can any devs help?

**Comment by Miyayes** [comment]
+1 https://www.reddit.com/r/brave_browser/comments/fksqsf/brave_sync_its_our_top_priority_and_were_working/fkzatpc/?context=3

**Comment by jDally987** [comment]
Discovered yesterday that as far as I can tell, most/all of my settings, custom search engines etc. all seem to have transferred, as well as the extensions’ STATE, but not the extensions themselves.

Once I copied over all the long gibberish folders (representing the chrome extension IDs) from my old User Data directory, they do show up in the browser and interestingly they’re all still signed in to my accounts specific to them + all associated data. e.g. Pushbullet was instantly ready to go, all signed in to my Google acct and still had the keyboard shortcuts I assigned it and everything.

**Comment by erotavlas-turbo** [comment]
> +1 https://www.reddit.com/r/brave_browser/comments/fksqsf/brave_sync_its_our_top_priority_and_were_working/fkzatpc/?context=3

Sync is not the same as profile copy. However, I hope to have it as soon as possible.

**Comment by bsclifton** [comment]
Sync v2 is coming really soon - but profile copying should work just fine. There are a few edge cases though

When copying the profile, first close out Brave (some of the DBs will put a lock on themselves when in use). I would recommend copying the ENTIRE `%USERPROFILE%\AppData\Local\BraveSoftware\Brave-Browser\User Data` folder instead of the child `Default` directory. The parent (`User Data`) folder has Chromium local state, extension state, and a lot of other stuff.

Where I mentioned there are some edge cases... If you move to a new PC, any of your saved passwords won't work. That's because they were encrypted with a device specific key (ex: Windows Crypt API, macOS keychain, etc) and while there are still entries in the SQLite DB, the actual password values can't be decrypted.  Everything else as I'm aware of it should mostly "work"

Given this was opened a while back, I'll close the issue out. Sorry that it took a really long time for a response - this issue definitely got lost in my notifications. But please re-open or continue to comment and I can help you out @erotavlasme

**Comment by erotavlasme** [comment]
@bsclifton I tried again with the latest version of brave 1.11.101 on windows 10 2004. Now the settings are maintained except the default search engine which is switched to google. The extensions are not present. So the problem is not solved.

Brave process is not executing in background. I tried both by coping %USERPROFILE%\AppData\Local\BraveSoftware\Brave-Browser\User Data or just %USERPROFILE%\AppData\Local\BraveSoftware\Brave-Browser\User Data\Default and the behaviour is the same. On GNU/linux it works perfectly by just copying Default.

**Comment by bsclifton** [comment]
@erotavlasme great- thanks for trying and confirming. Will try to reproduce again and report findings

**Comment by erotavlasme** [comment]
Hi,
after one year, I tried with latest version of brave 1.28.105 on windows 10 21H2. As with the previous version, settings are maintained except the default search engine which is switched to google and brave adblock mode (restored to default value). The extensions are not present. So the problem is not solved.

**Comment by erotavlasme** [comment]
Hi,
after a year, I tried with latest version of brave 1.40.113 on windows 10 21H2. Nothing new: settings are maintained except the default search engine which is switched to google, the sidebar and brave shield mode (restored to default value). The extensions are not present. So the problem is still not solved.
